### DISCUSSION TITLE
Error in Create Next App

### DISCUSSION THREAD

**Discussion by vladimir-nesterov** [vercel/next.js#53963] [Help]
### Summary

I start new project 
   `npx create-next-app@latest my-project`
I don't change anything, but I see this error in the browser console:

```
Warning: Prop `className` did not match. Server: "__className_20951f vsc-initialized" Client: "__className_20951f"
    at body
    at html
    at RedirectErrorBoundary (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/redirect-boundary.js:73:9)
    at RedirectBoundary (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/redirect-boundary.js:81:11)
    at NotFoundErrorBoundary (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/not-found-boundary.js:54:9) .......
```
Why did this happen? What can I do about it?

**Comment by ch00z00** [comment]
## Cause of this error

Are you using styled-components?
This is an error that styled-components manages and styles className, and this className does not match between the server and the front.

## Solution

However, the following 2 steps can be taken according to the [official Next.js docs](https://nextjs.org/docs/pages/building-your-application/styling#css-in-js)!

1. Install plugin
```
# npm
npm i babel-plugin-styled-components
#yarn
yarn add babel-plugin-styled-components
```

2. Add a .babelrc directly under the project folder as follows.
```
{
  "presets": ["next/babel"],
  "plugins": [["styled-components", { "ssr": true }]]
}
```

Try these & good luck! 🍀

**Comment by icyJoseph** [comment]
@vladimir-nesterov do you see the same error if you try this in an incognito browser tab? I suspect this is because of a browser extension. Do you see if if you use other browsers?
